src/main/java/org/isf/sms/providers/SmsSenderInterface.java

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
As noted above this is not the current license header.

Reply:
done